src/LabelSelector/LabelChip/LabelChip.test.tsx

```diff
@@ -77,4 +77,31 @@ describe("LabelChip", () => {
     // there shouldn't be a cancel icon
     expect(screen.queryByTestId("CancelIcon")).not.toBeInTheDocument();
   });
+  // test that the selected icon is rendered when selected is true
+  it("renders selected icon", () => {
+    render(<LabelChip label="Label" selected />);
+
+    // get the selected icon
+    const selectedIcon = screen.getByTestId("DoneIcon");
+
+    // check that the selected icon is rendered
+    expect(selectedIcon).toBeInTheDocument();
+  });
+  // renders tooltip on hover of label chip
+  it("renders tooltip on hover", async () => {
```

Comment:
One of the requirements is that no tooltip is rendered when there isn't a description. Can we add a test that captures this?

Reply:
added